packages/comby/comby.1.4.1/opam

```diff
@@ -0,0 +1,65 @@
+opam-version: "2.0"
+maintainer: "rvantonder@gmail.com"
+authors: "Rijnard van Tonder"
+homepage: "https://github.com/comby-tools/comby"
+bug-reports: "https://github.com/comby-tools/comby/issues"
+dev-repo: "git+https://github.com/comby-tools/comby.git"
+license: "Apache-2.0"
+build: [
+    [
+      "dune"
+      "build"
+      "-p"
+      name
+      "-j"
+      jobs
+      "@install"
+      "@runtest" {with-test}
+    ]
+]
+depends: [
+  "dune" {>= "2.7.0"}
+  "ocaml" {>= "4.08.1"}
+  "conf-autoconf"
+  "conf-gmp"
+  "conf-libev"
+  "conf-libpcre"
+  "conf-m4" {os = "linux"}
+  "conf-pkg-config"
+  "conf-sqlite3"
+  "conf-zlib" {os = "linux"}
+  "angstrom" {>= "0.15.0"}
+  "cohttp-lwt-unix"
+  "comby-kernel"
```

Comment:
```suggestion
  "comby-kernel" {=version}
```
We tend to have packages version stick together to avoid compatibility issues and simplify releases, is there a specific reason you would like to keep this unconstrained?

Reply:
Nope, no specific reason. I'll update this!